components/steering_wheel/HW/HW_intc.c

```diff
@@ -9,9 +9,9 @@
  *                             I N C L U D E S
  ******************************************************************************/
 
-#include "stm32f1xx.h"
 #include "HW_intc.h"
-
+#include "SystemConfig.h"

```

Comment:
why did you make these changes?

Reply:
This was for when it was interrupt driven, I forgot to remove, have just done it.